### ISSUE TITLE
There is no code

### ISSUE THREAD

**Issue by sshriver** [Dev-Team-2014/BonsaiBattles#1]
I'll admit I may be unfamiliar with this programming paradigm, but from the view of a novice it appears that this project contains nothing. While this will certainly be beneficial to the user due to a small footprint on their device storage upon installing, it will probably lead to a shallow game experience.

**Comment by Biode** [comment]
You're absolutely right. Who are you and why are you commenting on a 4 year old failed project?

**Comment by sshriver** [comment]
Scott, I forgot I had this account and noticed this repo. Sorry if the joke was annoying

**Comment by Biode** [comment]
oh hey scott. I too forgot about this. I wonder if anyone else is going to get email notifications about this convo.

**Comment by pickerj** [comment]
Wow. I too, err, three, forgot about this.

**Comment by ZachShea** [comment]
@sshriver I too agree that this could be a potential issue. We might need to start from scratch and rewrite the whole thing.

**Comment by sshriver** [comment]
@ZachShea woah, let's calm down there! Although this project may have some feature creep, I think we can trim it down without needing to discard a half a decade of work.

**Comment by ZachShea** [comment]
On a side note, you guys need profile photos, these shapeless blobs are not very friendly

**Comment by sshriver** [comment]
PULL REQUEST:
Added friendly.jpg

**Comment by atanasiom** [comment]
What’s up bros? It’s ya boiiiii
